test/hausdorff.jl

```diff
@@ -49,4 +49,30 @@ type_list = generate_test_types([Bool], [Gray])
         end
     end
     test_cross_type(dist, A, B, type_list)
+
+    dist_2 = GenericHausdorff(ImageDistances.MaxReduction(), ImageDistances.MaxReduction(), (1.0, 1.5))
+    dist_3 = GenericHausdorff(ImageDistances.MaxReduction(), ImageDistances.MaxReduction(), (1.0, 1.5, 2.0))
```

Comment:
What about exporting a new constructor:

```julia
AxisWeightedHausdorff(weights) = GenericHausdorff(MaxReduction(), MaxReduction(), weghts)
```

Reply:
This is a good idea, I'll add this constructor.